api/src/main/java/org/ccci/idm/user/dao/ldap/AbstractLdapUserDao.java

```diff
@@ -71,6 +72,7 @@
             .put(Attr.CONTACT, ImmutableSet.of(LDAP_ATTR_TELEPHONE, LDAP_ATTR_OBJECTCLASS))
             .put(Attr.CRU_PREFERRED_NAME, ImmutableSet.of(LDAP_ATTR_CRU_PREFERRED_NAME, LDAP_ATTR_OBJECTCLASS))
             .put(Attr.CRU_PROXY_ADDRESSES, ImmutableSet.of(LDAP_ATTR_CRU_PROXY_ADDRESSES, LDAP_ATTR_OBJECTCLASS))
+            .put(Attr.CRU_PASSWORD_HISTORY, ImmutableSet.of(LDAP_ATTR_CRU_PASSWORD_HISTORY, LDAP_ATTR_OBJECTCLASS))
```

Comment:
I would be fine with including `LDAP_ATTR_CRU_PASSWORD_HISTORY` in the `Attr.PASSWORD` set, that way all existing code will automatically save the password history when changing the password.